src/js/pages/__tests__/DeploymentsTab-test.js

```diff
@@ -16,7 +16,7 @@ const DCOSStore = require('../../stores/DCOSStore');
 const Deployment = require('../../structs/Deployment');
 const DeploymentsTab = require('../services/DeploymentsTab');
 const DeploymentsList = require('../../structs/DeploymentsList');
-const Service = require('../../structs/Service');
+const Application = require('../../structs/Application');
```

Comment:
The same should apply here. I will add it as a pending task to use also Pod instances here.